git-cms-merge-topic

```diff
@@ -71,19 +81,22 @@ while [ $# -gt 0 ]; do
       # - Pull request: `<pull-request-id>`
       GITHUB_USER=cms-sw ; 
       BRANCH=$1 ;
+      LOCAL_BRANCH=$1 ;
       case $1 in
         *:*)
           BRANCH=`echo $1 | cut -f2 -d:`
+          LOCAL_BRANCH=$BRANCH
           GITHUB_USER=`echo $1 | cut -f1 -d:` ;;
         [0-9]*) 
-          BRANCH=refs/pull/$1/head ;; 
+          BRANCH=refs/pull/$1/head 
+          LOCAL_BRANCH=PR$1 ;; 
```

Comment:
I would suggest `pull/$1` or `pr$1` (lowercase), but I guess it's a matter of personal taste